DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/DelegateGroups/DelegateGroupsViewModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models.DelegateGroups;
+
+    public class DelegateGroupsViewModel
+    {
+        public DelegateGroupsViewModel(IEnumerable<Group> groups)
+        {
+            DelegateGroups = groups.Select(g => new SearchableDelegateGroupViewModel(g));

```

Comment:
Yeah that is going to need to change once we start implementing the search/sort/filter functionality on that page. The helpers expect IEnumerable<T> where T : BaseSearchableItem (in the data project, see AdminUser for an example). We do all the filtering down then convert the resulting list to the list item view model and this logic lives in the constructor of the overall view model so that it can be unit tested more easily.